mon-pix/tests/unit/components/signup-form_test.js

```diff
@@ -146,4 +146,58 @@ module('Unit | Component | signup-form', function (hooks) {
       });
     });
   });
+
+  module('#onCguCheckboxChange', function () {
+    module('when checkbox is checked', function () {
+      test('it registers the cgu as accepted and reset validation status', function (assert) {
+        // given
+        const event = {
+          target: {
+            checked: true,
+          },
+        };
+        component.args.user = {
+          cgu: false,
+        };
```

Comment:
La valeur de la propriété `this.args.user.cgu` dépend uniquement de la valeur de `event.target.checked`, aussi il me semble que mettre une valeur initiale pour `component.args.user.cgu` rend la compréhension du test un tout petit peu plus difficile. Aussi je suggère la modification suivante : 
```suggestion
        component.args.user = {};
```